xprof/btx_aggreg_model.yaml

```diff
@@ -0,0 +1,135 @@
+:stream_classes:
+  - :name: interval_tally
```

Comment:
maybe we can call this stream class `aggreg`?

Reply:
Good point!

Reply:
Fixed by 6e21598